test/DGmethods/Euler/acousticwave-1d-imex.jl

```diff
@@ -0,0 +1,231 @@
+using CLIMA
+using CLIMA.Mesh.Topologies: StackedCubedSphereTopology, cubedshellwarp, grid1d
+using CLIMA.Mesh.Grids: DiscontinuousSpectralElementGrid
+using CLIMA.Mesh.Filters
+using CLIMA.DGmethods: DGModel, init_ode_state, VerticalDirection
+using CLIMA.DGmethods.NumericalFluxes: Rusanov, CentralGradPenalty,
+                                       CentralNumericalFluxDiffusive
+using CLIMA.ODESolvers: solve!, gettime
+using CLIMA.AdditiveRungeKuttaMethod
+using CLIMA.GeneralizedMinimalResidualSolver
+using CLIMA.ColumnwiseLUSolver: ManyColumnLU
+using CLIMA.VTK: writevtk, writepvtu
+using CLIMA.GenericCallbacks: EveryXWallTimeSeconds, EveryXSimulationSteps
+using CLIMA.PlanetParameters: planet_radius, day
+using CLIMA.MoistThermodynamics: air_density, soundspeed_air, internal_energy
+using CLIMA.Atmos: AtmosModel, SphericalOrientation,
+                   DryModel, NoRadiation, NoFluxBC,
+                   ConstantViscosityWithDivergence,
+                   vars_state, vars_aux,
+                   Gravity, HydrostaticState, IsothermalProfile,
+                   AtmosAcousticGravityLinearModel
+using CLIMA.VariableTemplates: flattenednames
+
+using MPI, Logging, StaticArrays, LinearAlgebra, Printf, Dates, Test
+const ArrayType = CLIMA.array_type()
+
+const output_vtk = false
+
+function main()
+  CLIMA.init()
+  mpicomm = MPI.COMM_WORLD
+
+  ll = uppercase(get(ENV, "JULIA_LOG_LEVEL", "INFO"))
+  loglevel = Dict("DEBUG" => Logging.Debug,
+                  "WARN"  => Logging.Warn,
+                  "ERROR" => Logging.Error,
+                  "INFO"  => Logging.Info)[ll]
+
+  logger_stream = MPI.Comm_rank(mpicomm) == 0 ? stderr : devnull
+  global_logger(ConsoleLogger(logger_stream, loglevel))
+
+  polynomialorder = 5
+  numelem_horz = 10
+  numelem_vert = 5
+
+  timeend = 60 * 60
+  # timeend = 33 * 60 * 60 # Full simulation
```

Comment:
What is the criteria for success in the full simulation?

Reply:
The first criterion is lack of `NaN`s :) The second one is visual inspection, you should see a wave that propagates similarly to Fig 3. in https://doi.org/10.1016/j.fluiddyn.2004.03.003.